tests/runtests.bats

```diff
@@ -84,7 +84,7 @@ cleanup() {
 @test "bring up with two disks" {
   export VAGRANT_CWD=tests/two_disks
   cleanup
-  tools/create_box_with_two_disks.sh ${VAGRANT_HOME} ${VAGRANT_CMD}
+  tools/create_box_with_two_disks.sh "${VAGRANT_HOME}" "${VAGRANT_CMD}"
```

Comment:
Did something flag these? Just wondering as it looks like something that I get reminded by shellcheck when I remember to run it, seems like we could benefit from maybe enabling it to run on the repo and fix up any issues as a separate PR?

Reply:
I remove this. When I run `bats ./test/runtests.sh`, I need to use the command `bundle exec vagrant` instead of `vagrant`.
Then this test fail because `bash` run the command `tools/create_box_with_two_disks.sh "${VAGRANT_HOME}" bundle exec vagrant` instead of `tools/create_box_with_two_disks.sh "${VAGRANT_HOME}" "bundle exec vagrant"`

Reply:
Ah, that would be handy to have, usually I'm running after doing `bundle --binstubs ...`, no harm if putting it up with a separate PR and we can try and put together something that runs shellcheck because I suspect it would have caught the issue you bumped into